wfl/Database/MongoDBManager.cs

```diff
@@ -19,69 +20,60 @@ public MongoDBManager()
             db = client.GetDatabase("WFL");
         }
 
-        public async Task<Restaurant> InsertRestaurantAsync(RestaurantDto r)
+        private string GenerateCollectionName<T>()
```

Comment:
Wouldn't the value change depending on the type supplied to this method? In some controller methods, I need to be able to access entities of different types. For instance, I need to be able to get a restaurant by id from the food orders controller to verify if the id is valid. There would be an issue if each instance of dbManager can only access one type of entity.